app/views/pages/routes/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<% set_page_title(title_with_error_prefix(t('page_titles.routes_show', position: page.position), false)) %>
+<% content_for :back_link, govuk_back_link_to(back_link_url, t('pages.go_to_your_questions')) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">
+          <span class="govuk-caption-l"><%= current_form.name %></span>
+          <%= t("page_titles.routes_show", position: page.position) %>
+      </h1>
+
+      <%= govuk_summary_list(actions: false) do |summary_list|
+          summary_list.with_row do |row|
+              row.with_key { t("page_route_card.question_title", position: page.position) }
+              row.with_value { "\"#{page.question_text}\"" }
+          end;
+      end %>
+
+      <% RouteSummaryCardDataService.call(page:, pages:).build_data.each do |card| %>
+          <%= govuk_summary_list(**card) %>
+      <% end %>
+
+      <%= govuk_button_link_to t("save_and_continue"), form_pages_path(current_form.id) %>
```

Comment:
It looks like the latest designs change this button to a normal link (see https://app.mural.co/t/gaap0347/m/gaap0347/1728478914347/6a7e0b709f50f00f81cab37414fdf44ec8601a70?wid=0-1730301452966)

```suggestion
      <%= govuk_link_to t("pages.go_to_your_questions"), form_pages_path(current_form.id) %>
```